src/lib/calc-helpers.js

```diff
@@ -39,3 +40,26 @@ export const setSliderByIndex = (idx, val) => {
   const identifier = match[1];
   calculator.setExpression({ id, latex: `${identifier}=${val}` });
 };
+
+/**
+ * saves the current graph and any gifs to local storage
+ * returns the new graph info to add to saved graph list
+ */
+export const saveCurrentGraph = async (name, frames, frameIDs) => {
+  const graph = calculator.getState();
+  const preview = await getImageData({
+    width: 160,
+    height: 160,
+    targetPixelRatio: 0.25
+  });
+
+  let newGraph = saveGraphToLocal(graph, name, preview, frames, frameIDs);
```

Comment:
Doesn't look like we need this assignment since we return the result on the next line. Think we can just:
```javascript
return saveGraphToLocal(graph, name, preview, frames, frameIDs);
```

Reply:
Done